tsconfig.json

```diff
@@ -3,12 +3,11 @@
 		"declaration": true,
 		"declarationMap": true,
 		"esModuleInterop": true,
-		"module": "ESNext",
-		"moduleResolution": "Bundler",
+		"module": "esnext",
+		"moduleResolution": "bundler",
```

Comment:
🤔 These come from https://github.com/JoshuaKGoldberg/create-typescript-app/pull/2060 <- https://github.com/JoshuaKGoldberg/zod-tsconfig, which uses a [`z.nativeEnum()`](https://zod.dev/?id=native-enums) on the inside. I don't know that there's a set practice in typescript-world around capitalizing these or not...

Reply:
I've found that VSCode always auto-suggests the capitalized versions.

Reply:
🙃 and TypeScript's docs are lowercase (ex https://www.typescriptlang.org/tsconfig/#module). Lovely.